DatadogInternal/Sources/NetworkInstrumentation/FirstPartyHosts.swift

```diff
@@ -27,7 +27,7 @@ public struct FirstPartyHosts: Equatable {
     public init(_ hosts: Set<String>) {
```

Comment:
The public API comment for this method is now outdated, it mentions `.datadog`, but not `.tracecontext`:

> Creates a `FirstPartyHosts` instance with the given set of host names by assigning `.datadog` header type to each.